pkg/cmd/image/inspect.go

```diff
@@ -195,5 +198,9 @@ func Inspect(ctx context.Context, client *containerd.Client, identifiers []strin
 		}
 	}
 
+	if len(errs) > 0 {
+		return fmt.Errorf("%d errors:\n%s", len(errs), strings.Join(errs, "\n"))
+	}
```

Comment:
I think you can still use `var errs []error` to store all the errors, and here do

```golang
return fmt.Errorf("%d errors:\n%w", len(errs), errors.Join(errs...))
```

When you print a `errors.Join`ed error, it will automatically print one error per line.